keps/sig-node/4800-cpumanager-split-l3cache/kep.yaml

```diff
@@ -0,0 +1,50 @@
+title: KEP Template
+kep-number: 4800
+authors:
+  - "@sphrasavath"
+  - "@wongchar"
+  - "@ajcaldelas"
+owning-sig: sig-node
+participating-sigs:
+  - sig-node
+status: implementable # provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
+creation-date: 2024-08-23
+reviewers:
```

Comment:
Please make sure to fill this out.

I can be a reviewer and maybe @mrunalp can be a tentative approver.

Reply:
Okay will add

Reply:
Who do I need to add as approvers?

Reply:
mrunalp 

And maybe you can add me and @ffromani as reviewers.

Reply:
yes please add me as reviewer. During the KEP planning on sig-node we may add more

Reply:
I saw on some other KEPs that they used @sig-node-leads as one of the approvers should I keep that? Also, I've added the reviewers and approvers

Reply:
What you have is fine for now.